cernopendata/modules/fixtures/data/records/cms-trigger-information-Run2012B-C.json

```diff
@@ -0,0 +1,28 @@
+[
```

Comment:
(Cosmetics: maybe also rename the file to `cms-trigger-information-2012.json` since we have all the runs there...)